app/controllers/course/video/submission/sessions_controller.rb

```diff
@@ -1,15 +1,23 @@
 # frozen_string_literal: true
 class Course::Video::Submission::SessionsController < Course::Video::Submission::Controller
+  load_and_authorize_resource :session, class: Course::Video::Session.name, through: :submission
+
+  def create
+    time_now = Time.zone.now
+    @session.session_start = time_now
```

Comment:
Shall we put the setting of `time_now` as a callback in the model instead? (Maybe using `after_build :foo, if: :new_record?`

Reply:
Yup, fixed.